mixer/pkg/perf/config.go

```diff
@@ -14,12 +14,16 @@
 
 package perf
 
+// TODO: We should ideally combine this file with pkg/server/Args. Unfortunately, pkg/serverArgs is not serializable.
```

Comment:
What does it take to make it serializable? Just adding the annotations?

Reply:
Annotations + Coming up with a strategy for serializing & reconciling adapters/templates during deserialization. They are currently kept as full-blown objects.